src/the_way/snippet.rs

```diff
@@ -164,6 +165,36 @@ impl Snippet {
         Ok(())
     }
 
+    /// Read potentially multiple snippets from a regular Gist (not a sync/backup Gist)
+    pub(crate) fn from_gist(
+        start_index: usize,
+        languages: &HashMap<String, Language>,
+        gist: &Gist,
+    ) -> color_eyre::Result<Vec<Self>> {
+        let mut index = start_index;
+        let mut snippets = Vec::new();
+        for (file_name, gist_file) in &gist.files {
+            let code = &gist_file.content;
+            let description = format!("{} - {}", gist.description, file_name);
+            let language = &gist_file.language;
+            let tags = format!("gist {}", gist.id);
```

Comment:
Could you change this to format!("gist:{}", gist.id)? Tags are space-separated in the CLI so the space would be confusing and incorrectly parsed after an edit. Also, tags are not uploaded to the synced Gist so it would be good to add the gist ID also to the description (to avoid uncredited copying of Gists)

Reply:
Just to clarify, my intention here was to have 2 separate tags `gist` and the ID e.g. `66fd3b1a529ed01794cff091e08898ab`. The `Snippet::new` thats called below here actually then calls `utils::split_tags(tags)` and splits the string into 2 tags.

Reply:
Ah I see. a unique tag doesn't seem very useful for searching so maybe it's better to only attach the "gist" tag and keep the ID solely in the description.